specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-01-20-preview/guestconfiguration.json

```diff
@@ -309,6 +309,100 @@
           "application/json"
         ]
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/providers/Microsoft.GuestConfiguration/software/{softwareId}": {
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

Would extension resource be cached in the future, please confirm with ARM team? Basically the pattern of exposing only GET without creating actual ARM resource, would that cause any problem. Are there other RP using this pattern?

Reply:
Thanks, I am checking this with some one from ARM team.